webpack/scenes/ContentViews/Create/CreateContentViewForm.js

```diff
@@ -29,13 +29,13 @@ const CreateContentViewForm = ({ setModalOpen }) => {
 
   useDeepCompareEffect(() => {
     const { id } = response;
-    if (id && status === STATUS.RESOLVED) {
+    if (id && status === STATUS.RESOLVED && saving) {
       setSaving(false);
       setRedirect(true);
     } else if (status === STATUS.ERROR) {
       setSaving(false);
     }
-  }, [response, status, error]);
+  }, [JSON.stringify(response), status, error, saving]);
```

Comment:
Now that we use `useDeepCompareEffect` we don't need to `JSON.stringify` anymore :)

Reply:
(You might need to rebase the PR to the latest master, I think that was just merged in.)

Reply:
I will reach out to do this! thanks